### ISSUE TITLE
Turn msg-extractor into an actual Python package

### ISSUE THREAD

**Issue by giampaolo** [mattgwwalker/msg-extractor#25]
Currently the recommended way to install this lib is via GIT and there's
no tarball on PYPI. This has different drawbacks:

- it is slow to download compared to a HTTP fetch from the PYPI repo
- it is error prone when it comes to pin a specific revision/cset in
requirements.txt (e.g. git+
https://github.com/mattgwwalker/msg-extractor.git@2a24c9950d34932ed5979693cb70d758d78715df#egg=ExtractMsg)
- e.g. I didn't understand why I coulndn't import ExtractMsg and it was
because I add to append #egg=ExtractMsg in requirements.txt
- a GIT url cannot be used as a install_requires=... dependency in
setup.py so you have to necessarily use pip + requirements.txt.
- relying on requirements.txt means that if package A depends from
msg-extractor and packages B and C depend from package A you'll have
to put msg-extractor in all 3 requirements.txt of pkg A, B and C
(whereas with setup.py you would only set it in setup.py of A)

Basically right now msg-extractor is a Python script hosted on github.
Instead it should benefit if it were aa standard Python package hosted on
PYPI.

Practical proposal:

- provide a setup.py, set package name to msgextractor for PEP8
compliance
- rename MsgExtractor.py to msgextractor.py
- API usage would be:

from msgextractor import Message
m = Message(...)

- cmdline usage would be

python -m msgextractor example.msg

If this is acceptable I can provide a PR.

**Comment by mattgwwalker** [comment]
Sounds excellent to me. If you’ve got the time to make that happen, I’d be
more than pleased.

Cheers,

Matthew

**Comment by giampaolo** [comment]
You know this is gonna break code though, right? Are you ok with that?

**Comment by TheElementalOfCreation** [comment]
Version 0.21.0 is out. If your issue is not fixed, please comment and I will reopen it.
